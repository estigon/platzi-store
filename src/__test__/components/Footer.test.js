import React from 'react';
import { mount, shallow } from 'enzyme';
import Footer from '../../components/Footer';
import { create } from 'react-test-renderer';

describe('<Footer />', () => {
    const footer = mount(<Footer />);
    const footer2 = shallow(<Footer />);

    test('Render del componente Footer', () => {

        // console.log("mount")
        console.log(footer.debug())
        // console.log("shallow")
        console.log(footer2.debug())
        expect(footer.length).toEqual(1);
    });

    test('Render del titulo', () => {
        expect( footer.find('.Footer-title').text()).toEqual('Platzi Store')
    });
});

describe('Snapshot <Footer />', () => {

    test('comprobar UI del elemento <Footer />', () => {
        //pasamos al create el elemento sin provider porque no tiene que ver con redux
        const footer = create(<Footer />)
        expect(footer.toJSON()).toMatchSnapshot()
    })
})