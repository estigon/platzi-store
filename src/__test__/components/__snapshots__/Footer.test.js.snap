// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot <Footer /> comprobar UI del elemento <Footer /> 1`] = `
<div
  className="Footer-wrapper"
>
  <p
    className="Footer-title"
  >
    Platzi Store
  </p>
  <p
    className="Footer-copy"
  >
    Todos los Izquierdos Reservados
  </p>
</div>
`;
